sigs.yml

```
@@ -0,0 +1,502 @@
+    meeting: Every other week on Wednesday at 08:00 PT
+    notes:
+      type: gDoc
+      value: 19bnXziPn2MZ9wO6684UoI4D-LCjGL5bTJkGhux29bx8
+    chat:
+      - type: slack
+        name: '#otel-prometheus-wg'
+        id: C01LSCJBXDZ
+    invites: calendar-prometheus
+    tcSponsors: []
+    gcLiaison: []
+  - name: Client Instrumentation
+    meeting: Every Tuesday at 9:00 AM PT
+    notes:
+      type: gDoc
+      value: 16Vsdh-DM72AfMg_FIt9yT9ExEWF4A_vRbQ3jRNBe09w
+    chat:
+      - type: slack
+        name: '#otel-client-side-telemetry'
+        id: C0239SYARD2
+    invites: calendar-client-side
+    tcSponsors: []
+    gcLiaison: []
+  - name: Kubernetes Operator
+    meeting: Every other week on Thursday at 09:00 PT
+    notes:
+      type: gDoc
+      value: 1Unbs2qp_j5kp8FfL_lRH-ld7i5EOQpsq0I4djkOOSL4
+    chat:
+      - type: slack
+        name: '#otel-operator'
+        id: C033BJ8BASU
+    invites: calendar-k8s-operator
+    tcSponsors: []
+    gcLiaison: []
+  - name: Community Demo Application
+    meeting: Every Wednesday at 8:00 PT
+    notes:
+      type: gDoc
+      value: 16f-JOjKzLgWxULRxY8TmpM_FjlI1sthvKurnqFz9x98
+    chat:
+      - type: slack
+        name: '#otel-community-demo'
+        id: C03B4CWV4DA
+    invites: calendar-demo-app
+    tcSponsors: []
+    gcLiaison:
+    - name: 'Austin Parker'
+      github: austinlparker
+  - name: Functions as a Service (FAAS)
+    meeting: Every other Tuesday at 12:00 PM PT
+    notes:
+      type: gDoc
+      value: 187XYoQcXQ9JxS_5v2wvZ0NEysaJ02xoOYNXj08pT0zc
+    chat:
+      - type: slack
+        name: '#otel-faas'
+        id: C04HVBETC9Z
+    invites: calendar-faas
+    tcSponsors: []
+    gcLiaison: []
+  - name: Profiling WG
+    meeting: Every other week on Thursday at 08:00 AM PT
+    notes:
+      type: gDoc
+      value: 19UqPPPlGE83N37MhS93uRlxsP1_wGxQ33Qv6CDHaEp0
+    chat:
+      - type: slack
+        name: '#otel-profiles'
+        id: C03J794L0BV
+    invites: calendar-profiling
+    tcSponsors: []
+    gcLiaison: []
+  - name: Security
+    meeting: Weekly on Wednesday at 08:30 AM PT
+    notes:
+      type: gDoc
+      value: 1P2xejC7lEkOV_Z-8E0oZPXLK5HOnUPNuRqKP0ZQ5fpg
+    chat:
+      - type: slack
+        name: '#otel-sig-security'
+        id: C05A85QC281
+    invites: calendar-sig-security
+    tcSponsors:
+    - name: Reiley Yang
+      github: reyang
+    gcLiaison:
+    - name: 'Juraci Paixão Kröhling'
+      github: jpkrohling
+  - name: OpenTelemetry on Mainframes
+    meeting: Weekly on Tuesday at 10:00 AM PT
+    notes:
+      type: gDoc
+      value: 14p-bpofozTL4n3jy6HZH_TKjoOXvog18G1HBRqq6liE
+    chat:
+      - type: slack
+        name: '#otel-mainframes'
+        id: C05PXDFTCPJ
+    invites: calendar-mainframe
+    tcSponsors: []
+    gcLiaison:
+      - name: Alolita Sharma
+        github: alolita
+      - name: Morgan McLean
+        github: mtwo
+      - name: Daniel Dyla
+        github: dyladan
```

```suggestion
    gcLiaison:
      - name: Alolita Sharma
        github: alolita
      - name: Morgan McLean
        github: mtwo
      - name: Daniel Dyla
        github: dyladan
```

@alolita @mtwo @dyladan all of you were listed as sponsors for this SIG, can one of you assign yourself to be the main liaison person for this SIG?

---

I'm attending the most regularly, I'll do it